plugins/modules/proxysql_mysql_users.py

```diff
@@ -177,6 +190,21 @@ def load_config_to_runtime(cursor):
     return True
 
 
+def mysql_native_password(cleartext_password):
```

Comment:
just a nitpick, but we could also rename this function to `_mysql_native_password` to indicate it shouldn't be called directly (it should always be called via `encrypt_cleartext_password`)

Reply:
Yes, I agree. Done.